packages/i18next/README.md

```diff
@@ -44,10 +44,111 @@ npm install @sapphire/plugin-i18next @sapphire/framework discord.js
 
 ## Usage
 
+This registers the necessary methods and options in the Sapphire client to be able to use the message translations.
+
 ```typescript
+// Main bot file
```

Comment:
Make this mention that it should be done before logging in 🙏

Reply:
done. 📮

Reply:
Where

Reply:
I'm blind